src/containers/genePage/genomeFeatureViewer.js

```diff
@@ -100,8 +107,8 @@ class GenomeFeatureViewer extends Component {
                                                                  url={this.externalJBrowseUrl}
                                                                  width={this.props.width}
                                                   /> : ''}
-              {this.state.loadState == 'error' ? 'Unable to load' : ''}
             </a>
+            {this.state.loadState == 'error' ? <div className={style.error}>Unable to retrieve data</div> : ''}
```

Comment:
Maybe consider using bootstrap's `text-danger` class here instead of defining your own to try and keep style consistent?